scripting/viewmodel_override.sp

```diff
@@ -187,6 +189,71 @@ void UpdateClientWeaponModel(int client) {
 		
 		bitsActiveModels |= MODEL_WORLD_ACTIVE;
 	}
+
+	if(bitsActiveModels & (MODEL_VIEW_ACTIVE|MODEL_WORLD_ACTIVE) != 0) {
+		char uid[64];
+		if(CWX_GetItemUIDFromEntity(weapon, uid, sizeof(uid)) && CWX_IsItemUIDValid(uid)) {
+			if(bitsActiveModels & MODEL_VIEW_ACTIVE != 0) {
+				KeyValues viewmodel_offset = CWX_GetItemExtData(uid, "viewmodel override offset");
+				if(viewmodel_offset) {
+					int weaponvm = EntRefToEntIndex(g_iLastViewmodelRef[client]);
```

Comment:
You should be able to skip `EntRefToEntIndex` &mdash; everything here should be able to take the `g_iLastViewmodelRef[client]` entity reference directly, though you can do `int weaponvm = g_iLastViewmodelRef[client]` so you don't have to write the global out every time.